performanceplatform/client/base.py

```diff
@@ -115,8 +115,10 @@ def _request(self, method, path, data=None):
             except:
                 log.error('[PP-C] {}'.format(response.text))
                 raise
-
-            json = response.json()
+            try:
+                json = response.json()
+            except ValueError:
```

Comment:
I think the signon convention is wrong. If you're doing to return an empty
response, then 204 No Content is appropriate.

I'll update stagecraft, and then update the client to only try to parse JSON if it gets
back status code associated with a body.